staubli_rx160_gazebo/urdf/rx160.xacro

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?>
+<robot name="staubli_rx160_gazebo" xmlns:xacro="http://ros.org/wiki/xacro">
```

Comment:
I did have name clashes whilst developing the package, hence the suffix.
I'll try without it, and if it works, it makes sense to remove so that it follows the standard adopted in the other packages.